apps/simple-trading-app/src/app/components/stepper/stepper.tsx

```diff
@@ -1,93 +1,143 @@
   disabled?: boolean;
 };
 
-interface StepperProps {
-  steps: Step[];
+export interface StepperProps {
+  steps: TStep[];
+  className?: string;
 }
 
-export default ({ steps }: StepperProps) => {
+export const Stepper = ({ steps }: StepperProps) => {
   const [activeStep, setActiveStep] = React.useState(0);
+  const lastStep = steps.length - 1;
+  const isLastStep = activeStep === lastStep;
 
   const handleClick = (index: typeof activeStep) => {
     setActiveStep(index);
   };
 
-  const handleNext = () => {
-    setActiveStep((prevActiveStep) => prevActiveStep + 1);
-  };
-
-  const handleBack = () => {
-    setActiveStep((prevActiveStep) => prevActiveStep - 1);
+  const handleKeyPress = (
+    event: React.KeyboardEvent<HTMLButtonElement>,
+  ) => {
+    if (event.key === 'ArrowLeft' && activeStep > 0) {
+      return setActiveStep((prevActiveStep) => prevActiveStep - 1);
+    } else if (event.key === 'ArrowRight' && !isLastStep) {
+      return setActiveStep((prevActiveStep) => prevActiveStep + 1);
+    }
+    return false;
   };
 
-  const handleReset = () => {
-    setActiveStep(0);
+  const handleNext = () => {
+    if (!isLastStep) {
+      setActiveStep((prevActiveStep) => prevActiveStep + 1);
+    }
   };
 
   return (
-    <Paper square elevation={1} sx={{ p: 3, mt: 5 }}>
-      <Stepper activeStep={activeStep} orientation="vertical">
-        {steps.map((step, index) => (
-          <Step key={step.label}>
-            <StepLabel
-              optional={
-                index === steps.length - 1 ? (
-                  <Typography variant="caption">Last step</Typography>
-                ) : null
-              }
-              style={{ cursor: 'pointer' }}
-              onClick={() => handleClick(index)}
+    <div>
+      <ol
+        aria-label={t('Step by step to make a trade')}
+        className="relative flex md:flex-col justify-between list-none"
+      >
+        {steps.map((step, index) => {
+          const isActive = activeStep === index;
+          const isFirstStep = !index;
+          const isLastStep = lastStep === index;
+          return (
+            <li
+              className="flex-1"
+              key={`${index}-${step.label}`}
+              aria-label={t(`Step ${index + 1}`)}
             >
-              {step.label}
-            </StepLabel>
-            <StepContent>
-              <Typography sx={{ mb: 2 }}>{step.description}</Typography>
-              {step.component}
-              <Divider sx={{ mb: 2 }} />
-              <Box>
-                <div>
-                  <Button
-                    variant="secondary"
-                    onClick={handleNext}
-                    disabled={step.disabled}
+              <div className="flex relative md:pt-16">
+                {!isFirstStep ? (
+                  <div
+                    aria-hidden
+                    className="flex-auto	absolute top-[20px] -left-1/2 right-1/2 md:-top-1/2 md:bottom-1/2 md:left-[29.5px] md:right-auto"
```

Comment:
Chore (non-blocking): Remove unnecessary text gap here and on line 78

Reply:
Done